### DISCUSSION TITLE
Integration with PenPot

### DISCUSSION THREAD

**Discussion by gvlx** [phosphor-icons/homepage#330] [Q&A] [answered]
Hi,

Is there a way to integrate directly with PenPot https://penpot.app/ ?

Thank you.

**Comment by rektdeckard** [comment]
It looks like PenPot does not have a plugin ecosystem yet, but it is something they are working on. We will probably wait until that happens before we investigate. The alternative would be to create and maintain a community file, which is quite a bit more manual effort.

Do you have any issues pasting/importing Phosphor icons into PenPot?

**Reply by rektdeckard** [reply]
Update: Penpot recently announced a plugin system that is currently in closed beta, and we have been invited to test it out. I'm happy to say we have a working Phosphor plugin, which hopefully will be available to the public very soon:

https://github.com/phosphor-icons/penpot

**Reply by rektdeckard** [reply, marked as answer]
Penpot plugins are officially 1.0 now, and ours is publicly available. Here is an [installation deeplink](https://design.penpot.app/#?plugin=https://phosphor-icons.github.io/penpot/manifest.json) if you need it.
